scripts/appscript.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -eu -o pipefail
+
+[[ $# -lt 1 ]] && {
+    echo "Usage $0 <BUILD_SLUG>" >&2
+    echo "  Example: $0 bucket-foo/randomid" >&2
+    exit 1
+}
+
+# Required vars
+BUILD_SLUG=$1
+VAULT_VERSION=$2
+DYNAMODB_TABLE=$3
+KSM_KEY_ID=$4
+SSM_PATH=$5
+
+# Internal vars
+AWS_AZ=$(curl -sSL http://169.254.169.254/latest/meta-data/placement/availability-zone)
+AWS_DOMAIN=$(curl -sSL http://169.254.169.254/latest/meta-data/services/domain)
+SALT_SLUG="${BUILD_SLUG}/salt"
+
+# Export standard aws envs
+export AWS_DEFAULT_REGION=${AWS_AZ:0:${#AWS_AZ} - 1}
+
+# Export Vault local address
+export VAULT_ADDR=http://127.0.0.1:8200
+
+#Update salt-call to
+sudo yum update salt-master
```

Comment:
I'm not seeing anything here that requires a salt-master? Is there some vault module using functionality of the master?

Reply:
I was playing around with the salt...it's not required anywhere